Stripe/PublicHeaders/STPSourceInfoViewController.h

```diff
@@ -0,0 +1,59 @@
+//
+//  STPSourceInfoViewController.h
+//  Stripe
+//
+//  Created by Ben Guo on 2/9/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import <Stripe/Stripe.h>
+#import "STPCoreTableViewController.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol STPSourceInfoViewControllerDelegate;
+@class STPSourceParams;
+
+/** You can use this view controller to collect information for payment sources that require additional information upon creation (e.g. iDEAL, which requires your customer's name and bank.) Note that this view controller renders a right bar button item that submits the form, so it must be shown inside a `UINavigationController`.
+ */
+@interface STPSourceInfoViewController : STPCoreTableViewController
+
+/**
+ *  Initializes a new `STPSourceInfoViewController` with the provided parameters. When the user submits the form, the view controller will return a copy of the provided `STPSourceParams` object to its delegate, updated with the information entered by the user.
+ *
+ *  @param sourceParams     The source parameters to collect additional info for. @see STPSourceParams
+  *  @param theme           The theme to use to inform the view controller's visual appearance. @see STPTheme
+ */
+- (nullable instancetype)initWithSourceParams:(STPSourceParams *)sourceParams
```

Comment:
I think it'd be best if you could immediately use the `STPSourceParams` object returned by this view controller, so it'd have to be prefilled with amount and returnURL, in addition to things in `prefilledInformation`. Maybe to avoid the conflicting information issue, we should just have an initializer that takes a source type, amount, returnURL, and prefilledInformation?